malta.ld

```diff
@@ -26,7 +26,7 @@ SECTIONS
     . = ALIGN(4096);
   } : text
 
-  .data ADDR(.text) + SIZEOF (.text):
+  .data :
```

Comment:
For the same reason empty linker sets will never work correctly - but I don't think we should worry about that.